src/jbmc/jbmc_parse_options.cpp

```diff
@@ -625,14 +641,44 @@ int jbmc_parse_optionst::get_goto_program(
   return -1; // no error, continue
 }
 
-bool jbmc_parse_optionst::process_goto_program(
-  const optionst &options)
+void jbmc_parse_optionst::process_goto_function(
+  const irep_idt &function_name,
+  goto_functionst::goto_functiont &function,
```

Comment:
should pass `goto_programt`

Reply:
See discussion above about various passes that operate on `goto_functiont`. This change should be made as part of any PR that addresses those instances.